### ISSUE TITLE
[Feature Request]: Include "Company" to the SCIM supported mappings

### ISSUE THREAD

**Issue by timon-michel-scopevisio** [grokability/snipe-it#15437]
### Is your feature request related to a problem? Please describe.

We have the multi-tenant support as well as SCIM synchronization enabled. If we synchronize a new user, it is not visible to any admin except the super admin. (which we try to reduce usage of)

### Describe the solution you'd like

Add the "Company" field to the list of supported synchronization attributes

### Describe alternatives you've considered

Just changing the Company manually: Works, but i need to login as super-admin

**Comment by triangleicon** [comment]
This feature would be very helpful to have when using multi company. I have to manually and change the user company after they are synced from entra, which can get a bit tedious.

**Comment by jbagot** [comment]
Agree, we need the same feature, we have a multi company snipeit and would be nice to have it. For now, what we did is to add the company to another field (like url or address) and then we have an script that move this field to the company field using the API. It's an ugly workaround but works, we did the same in with location.

**Comment by snipe** [comment]
We actually include companies (plural) in SCIM syncing now, so I’ll close this for now, but open a new issue if the most recent version of Snpe-IT doesn’t meet your needs.
